mk/cargo.sh

```diff
@@ -99,6 +100,12 @@ case $target in
     export CARGO_TARGET_MIPSEL_UNKNOWN_LINUX_GNU_LINKER=mipsel-linux-gnu-gcc
     export CARGO_TARGET_MIPSEL_UNKNOWN_LINUX_GNU_RUNNER="$qemu_mipsel"
     ;;
+  riscv64gc-unknown-linux-gnu)
+    export CC_riscv64gc_unknown_linux_gnu=riscv64-linux-gnu-gcc
```

Comment:
Why are we using GCC here instead of clang? I generally prefer to use clang unless we have a good reason not to.

Reply:
Changed to clang in 71dc9e9